examples/vue-notification-center-example/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@novu/notification-center-vue": "^0.10.1",
-    "vue": "^3.2.41"
+    "vue-demi": "^0.13.11"
```

Comment:
I don't think that we should use the `vue-demi` here in the example app. Please also remove any other changes, like in other files too.

Reply:
i thought it would be better since im also going to be creating an example app in vue 2 so it would showcase the usage of vue-demi

Reply:
As far as I understood this was that we would use the `vue-demi` only in the `notification-center` package, but the app that will have a dependency on `notification-center` will use the `vue 2` or `vue 3`. Otherwise, it doesn't make sense, bcos our users will be forced to use `vue-demi`.

Reply:
ok no problem